cornice/ext/wtforms.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+def validate_wtforms_schema(form_schema, with_matchdict=False):
+    """Validates JSON data through WTForms schema.
+
+    For this function to properly work you have to install wtforms_json
+    package and add this code::
+
+        import wtforms_json
+        wtforms_json.init()
+
+    to the module where you define WTForms schemas, just after WTForms
+    imports.
+
+    Here's an example usage::
+
+        @view(
+            ...
+            validators=( validate_wtforms_schema(MyFormSchema), ),
+            ...
+        )
+
+    This function also has a keyword argument *with_matchdict* set
```

Comment:
Rather than defining the existance and use of this parameter in the docstring like that, we try to use the `:param with_matchdict` syntax. See https://github.com/mozilla-services/cornice/blob/master/cornice/service.py#L51-L52 for an example.